app/models/order.rb

```diff
@@ -15,4 +17,25 @@ class Order < ApplicationRecord
     Table5: 5,
     Table6: 6
   }
+
+  def total
```

Comment:
calculates the total of the order by summing the associated order_items.  Multiplies each order_items price by its quantity